src/Modal.js

```diff
@@ -156,6 +160,19 @@ const Modal = React.createClass({
     if (this.props.keyboard && e.keyCode === 27) {
       this.props.onRequestHide();
     }
+  },
+
+  focusModalContent () {
+    this.lastFocus = document.activeElement;
```

Comment:
As far as I can tell yes it is in relation to iFrames. I wasn't even really aware of that property and why to use it till I read up on it this weekend. So thanks @taion for steering us in a good direction.